app/pages/project/about/index.jsx

```diff
@@ -31,16 +27,31 @@ class AboutProject extends Component {
     this.getPages();
   }
 
-  constructPagesData(apiResponse) {
+  componentDidUpdate(prevProps) {
+    const prevTranslations = prevProps.translations.strings.project_page;
+    if (prevTranslations !== this.props.translations.strings.project_page) {
+      this.setState({
+        pages: this.constructPagesData()
+      });
+    }
+  }
+
+  constructPagesData() {
     const availablePages = [];
 
     for (const url_key in SLUG_MAP) {
-      const matchingPage = apiResponse.find(page => page.url_key === url_key);
-      if (matchingPage && matchingPage.content && matchingPage.content !== '') {
+      const { pages, translations } = this.props;
+      const matchingPage = pages.find(page => page.url_key === url_key);
+      const pageTranslations = translations ? translations.strings.project_page : [];
+      const [matchingTranslation] = pageTranslations.filter(page => page.translated_id === parseInt(matchingPage.id));
+      const { content } = (matchingTranslation && matchingTranslation.strings) ?
+        matchingTranslation.strings :
+        matchingPage;
+      if (content && content !== '') {
```

Comment:
An empty string is false, so wouldn't the second expression be logically true if the first is?

Reply:
I was lazy and just copied the logic from your original code but I thought an empty string is logically true.

Reply:
<img width="211" alt="screenshot 2017-12-11 15 16 35" src="https://user-images.githubusercontent.com/2725841/33838080-513afb34-de86-11e7-9463-cd7ed1179b7a.png">

Reply:
Interesting.
`if (matchingPage && matchingPage.content && matchingPage.content !== '')`